internal/web/context.go

```diff
@@ -0,0 +1,22 @@
+package web
+
+import (
+	"github.com/labstack/echo/v4"
+	"go.uber.org/zap"
+)
```

Comment:
Using context.Context directly as a field in AppContext could lead to unexpected behavior. Consider embedding `echo.Context` or ensuring you handle it properly to avoid potential method conflicts or unintended behavior when interacting with echo.Context.